ja/lessons/specifics/nerves.md

```diff
@@ -110,24 +112,34 @@ Elapsed time: 8.022 s
 
 ## ネットワークの設定
 
-次のステップはネットワークを設定することです。Nervesのエコシステムはさまざまなパッケージを提供しています。[nerves_network](https://github.com/nerves-project/nerves_network) は、有線イーサネットポートを介してデバイスをネットワークに接続するために必要なものです。
+次のステップはネットワークを設定することです。Nervesのエコシステムはさまざまなパッケージを提供しています。[vintage_net](https://github.com/nerves-networking/vintage_net) は、有線イーサネットポートを介してデバイスをネットワークに接続するために必要なものです。
 
-あなたのプロジェクトにはすでに `nerves_init_gadget` が依存関係として存在しています。しかし、デフォルトではDHCPを使います（ `config/config.exs` の中の `config :nerves_init_gadget` の後の実行内容を確認してください）。静的IPアドレスを持つ方が簡単です。
+あなたのプロジェクトにはすでに [`nerves_pack`](https://github.com/nerves-project/nerves_pack) が依存関係として存在しています。しかし、デフォルトではDHCPを使います（ config/target.exs の中の config :vintage_net の後の設定内容を確認してください）。静的IPアドレスを持つ方が簡単です。
 
-静的ネットワークを設定するには、 `config/config.exs` に以下の行を追加する必要があります:
+静的ネットワークを設定するには、 `config/target.exs` 内の `:vintage_net` 設定を更新する必要があります:
 
-```
+```elixir
 # Statically assign an address
-config :nerves_network, :default,
-  eth0: [
-    ipv4_address_method: :static,
-    ipv4_address: "192.168.88.2",
-    ipv4_subnet_mask: "255.255.255.0",
-    nameservers: ["8.8.8.8", "8.8.4.4"]
+config :vintage_net,
+  regulatory_domain: "US",
+  config: [
+    {"usb0", %{type: VintageNetDirect}},
+    {"eth0",
+     %{
+       type: VintageNetEthernet,
+       ipv4: %{
+         method: :static,
+         address: "192.168.88.2",
+         prefix_length: 24,
+         gateway: "192.168.88.1",
+         name_servers: ["8.8.8.8", "8.8.4.4"]
+       }
+     }},
+    {"wlan0", %{type: VintageNetWiFi}}
   ]
 ```
 
-この設定は有線接続用です。無線接続を使用したい場合は、[Nerves network documentation](https://github.com/nerves-project/nerves_network#wifi-networking) を参照してください。
+この設定は有線接続用です。Wi-Fiを使用したい場合は、 [VintageNet Cookbook](https://hexdocs.pm/vintage_net/cookbook.html#wifi) を参照してください。
```

Comment:
```suggestion
この設定は有線接続の設定のみを更新します。WiFiを使用したい場合は、 [VintageNet Cookbook](https://hexdocs.pm/vintage_net/cookbook.html#wifi) を参照してください。
```

Reply:
I think "WiFi" is better than "Wi-Fi" as it is used in the Nerves documentation.